tests/unit/amo/reducers/test_reviews.js

```diff
@@ -570,13 +572,27 @@ describe(__filename, () => {
 
       const state = reviewsReducer(
         undefined,
-        showEditReviewForm({
-          reviewId: review.id,
-        }),
+        showEditReviewForm({ reviewId: review.id }),
       );
 
       expect(state.view[review.id].editingReview).toEqual(true);
     });
+
+    it('exits beginDeleteAddonReview state', () => {
+      const review = { ...fakeReview, id: 837 };
```

Comment:
Nit: This applies to most tests added to this file: You don't really need to set a specific `id` for the `fakeReview` because you are never referring to that `id` directly. You just use `review.id` throughout the test, so it's unnecessary for the test to "know" what the `id` is.

Reply:
All of those tests do rely on `review.id` for things like when they check `state.view[review.id].editingReview`. Sometimes `fakeReview` is passed in as a default value to helpers so I thought it would be safer to give each one a unique ID. I guess it's not so important.